ngrinder-controller/src/main/java/org/ngrinder/infra/config/WebMvcConfig.java

```diff
@@ -122,4 +125,13 @@ public void addResourceHandlers(ResourceHandlerRegistry registry) {
 			, "/**/*.woff" , "/**/*.ttf"};
 		registry.addResourceHandler(staticPathPatterns).addResourceLocations(this.resourceProperties.getStaticLocations()).setCachePeriod(3600);
 	}
+
+	@Bean
+	public UncheckedObjectMapper objectMapper() {
```

Comment:
If return type is `UncheckedObjectMapper` , when you change this implementation, you should modified all `@Autowired` field type.

how about `public ObjectMapper objectMapper()`?